Resources/translations/jsmessages.nl.yml

```diff
@@ -0,0 +1,55 @@
+pim_menu:
+    tab.reference_data: Verwijzings gegevens
+    item.reference_data: Verwijzings gegevens
+    navigation.reference_data: Verwijzings gegevens
+
+pim_custom_entity:
+    index_title: Overzicht
+    create_popin.title: Maken
+    form:
+        field.label.code: Code
+        tab:
+            history.title: Geschiedenis
+            properties:
+                title: Eigenschappen
+                section:
+                    common: Algemeen
+                    label_translations: Labels
+    button:
+        create: Maken
+    message:
+        created: Succesvol nieuwe Verwijzingsgegevens aangemaakt
+        removed: Verwijzingsgegevens met succes verwijderd
+    error:
+        removing: De referentiegegevens konden niet worden verwijderd.
+    export.csv.entity_name:
+        label: De Verwijzings gegevens naam
+        help: De Verwijzings gegevens naam die u wilt importeren
+        no_reference_data: Geen Verwijzings gegevens beschikbaar
+    import.csv.entity_name:
+        label: Verwijzings gegevens naam
+        help: De Verwijzings gegevens naam die u wilt importeren
+        no_reference_data: Geen Verwijzings gegevens beschikbaar
```

Comment:
Instead of 'Verwijzings gegevens' or 'referentiegegevens' you could use 'Referentie data'